src/internal/operators/skipUntil.ts

```diff
@@ -57,7 +57,9 @@ class SkipUntilSubscriber<T, R> extends OuterSubscriber<T, R> {
              outerIndex: number, innerIndex: number,
              innerSub: InnerSubscriber<T, R>): void {
     this.hasValue = true;
-    this.innerSubscription.unsubscribe();
+    if(this.innerSubscription) {
```

Comment:
This is just nitpicking but in all code around here there's always space between `if` and the starting parenthesis `if (this.innerSubscription)`.

Reply:
fixed